src/components/calculator/index.js

```diff
@@ -12,56 +11,94 @@ export const packEV = async (set) => {
+        const commonReturn = {
+            name,
+            price: prices.usd,
+            rarity,
+            type: type_line,
+            images: image_uris ? image_uris : card_faces[0].image_uris
+        };
+        if (type_line.toLowerCase().includes('basic') || type_line.toLowerCase().includes('gate')) {
+            return { ...commonReturn, probability: probOfLand }
+        } else if (rarity === 'common') {
+            return { ...commonReturn, probability: probOfComm }
         } else if (rarity === 'uncommon') {
-            return { name, price: prices.usd, rarity, probability: probOfUncomm }
+            return { ...commonReturn, probability: probOfUncomm }
         } else if (rarity === 'rare') {
-            return { name, price: prices.usd, rarity, probability: probOfRare }
+            return { ...commonReturn, probability: probOfRare }
+        } else if (rarity === 'mythic') {
+            return { ...commonReturn, probability: probOfMythic }
         } else {
-            return { name, price: prices.usd, rarity, probability: probOfMythic }
+            return { ...commonReturn, probability: probOfOther }
         }
     })
 
+    //calcing pack value including all cards in set
     let packValue = 0;
     let cardValue = 0;
     for (let card of newSetData) {
         cardValue = card.price * card.probability;
         packValue += cardValue;
     }
 
-    console.log(newSetData);
+    //numPacks will be variable based upon the set later
+    let numPacks = 36;
+    const boxValue = packValue * numPacks;
+
+    //make seperate "valuelist" that excludes cards with price < $2
+    const valueSetData = newSetData.filter(card => parseFloat(card.price) >= 2);
+
+    //calcing pack value excluding cards < $2
+    let filteredPackValue = 0;
+    let filteredCardValue = 0;
+    for (let card of valueSetData) {
+        filteredCardValue = card.price * card.probability;
+        filteredPackValue += filteredCardValue;
+    }
+
+    const filteredBoxValue = filteredPackValue * numPacks;
+
+    const setEVData = {
+        packValue: packValue.toFixed(2),
+        boxValue: boxValue.toFixed(2),
+        filteredPackValue: filteredPackValue.toFixed(2),
+        filteredBoxValue: filteredBoxValue.toFixed(2),
+        totalSetData: newSetData,
+        valueSetData: valueSetData,
+    };
+
+
+
     //if set is before shards of alara (no mythics yet) then
     //  nested - if (set === LEA(alpha) or LEB(beta) or 2ED(unlimited)) find probabilities for cards > $2
     //      note - alpha(p(rare-land)=.0413 p(uncom-land)=.215 p(comm-land)=38.84)
     //  nested - else (find probabilities for cards > $2) 
     //else if set is before battle for zendi then find probs for cards > $2 note - %12.5 for pack to have myth insted of rare
     //else find probs for cards > $2 note - %13.51 for pack to have myth insted of rare
-    return packValue
+    return setEVData
 }
```

Comment:
Overall I think this setEV function could be broken down into multiple functions. You may find a need to display a sets probabilities for each 'slot' on the set page for instance, and you could use the setRarityProbabilities() func to do that from a component.